src/arcaflow_plugin_sdk/schema.py

```diff
@@ -1490,7 +1513,9 @@ class Units:
     multipliers: typing.Annotated[
         Optional[Dict[int, Unit]],
         _name("Multipliers"),
-        _description("A set of multiplies that describe multiple units of scale."),
+        _description(
+            "A set of multiplies that describe multiple units of scale."
```

Comment:
I really don't want to get into nit-picking text you didn't change, but, just in case you stop by this neighborhood again and feel like spending an extra moment, "A set of multipliers" 😁

Reply:
i got you